src/unix/bsd/apple/b64/aarch64.rs

```diff
@@ -14,4 +20,11 @@ s! {
         pub __cpsr: u32,
         pub __pad: u32,
     }
+
+    #[repr(align(16))]
```

Comment:
This should be in `align.rs`, see other definitions.

Reply:
Should I then move everything in `aarch64.rs` over to `align.rs` and guard it with a `cfg_if ... target_arch = "aarch64"`?

Reply:
I think:
- b64/aarch64/mod.rs: just contain `cfg_if` stuff for `align.rs` for now. Having mod.rs makes sense when we add more stuff for AArch64 target (i.e. Apple Silicon) in the future.
- b64/aarch64/align.rs: Should have everything in the current `aarch64.rs`, as you said.

mod.rs should be:
```rust
cfg_if! {
    if #[cfg(libc_align)] {
        mod align;
        pub use self::align::*;
    }
}
```